queue_services/entity-filer/src/entity_filer/filing_processors/conversion.py

```diff
@@ -165,6 +181,155 @@ def _process_firms_conversion(
             )
 
 
+def _update_partner_change(
+        legal_entity: LegalEntity,
+        change_filing_rec: Filing,
+        change_filing: Dict,
+        filing_meta: FilingMeta,
+):
+    name_request = dpath.util.get(change_filing, "/conversion/nameRequest", default=None)
+    if name_request and (to_legal_name := name_request.get("legalName")):
+        alternate_name = AlternateName.find_by_identifier(legal_entity.identifier)
+        parties_dict = dpath.util.get(change_filing, "/conversion/parties")
+
+        legal_entity.legal_name = get_partnership_name(parties_dict)
+
+        legal_entity.alternate_names.remove(alternate_name)
+        alternate_name.end_date = change_filing_rec.effective_date
+        alternate_name.change_filing_id = change_filing_rec.id
+        # alternate_name.delete()
+        db.session.add(alternate_name)
+        db.session.commit()
+        db.session.delete(alternate_name)
+        db.session.commit()
+
+
+        new_alternate_name = AlternateName(
+            bn15=alternate_name.bn15,
+            change_filing_id=change_filing_rec.id,
+            end_date=None,
+            identifier=legal_entity.identifier,
+            name=to_legal_name,
+            name_type=AlternateName.NameType.OPERATING,
+            start_date=alternate_name.start_date,
+            registration_date=change_filing_rec.effective_date,
+        )
+        legal_entity.alternate_names.append(new_alternate_name)
+
+        filing_meta.conversion = {
+            **filing_meta.conversion,
+            "fromLegalName": alternate_name.name,
+            "toLegalName": to_legal_name,
+        }
+
+    # Update Nature of LegalEntity
+    if (
+        naics := change_filing.get("conversion", {})
+        .get("business", {})
+        .get("naics")
+    ) and (naics_code := naics.get("naicsCode")):
+        if legal_entity.naics_code != naics_code:
+            filing_meta.conversion = {
+                **filing_meta.conversion,
+                **{
+                    "fromNaicsCode": legal_entity.naics_code,
+                    "toNaicsCode": naics_code,
+                    "naicsDescription": naics.get("naicsDescription"),
+                },
+            }
+            legal_entity_info.update_naics_info(legal_entity, naics)
+
+
+def _update_person_change(
+        legal_entity: LegalEntity,
+        change_filing_rec: Filing,
+        change_filing: Dict,
+        filing_meta: FilingMeta,
+):
+    name_request = dpath.util.get(change_filing, "/conversion/nameRequest", default=None)
+    identifier = dpath.util.get(change_filing_rec.filing_json, "filing/business/identifier")
```

Comment:
@argush3 I am not sure how json going to be. This is the same assumption as we have in changeOfRegistration that we receive identifier of SP (not the person) from alternateName

Reply:
We can leave the same as until we get things sorted out.  Will cut tickets once we know how to move forward.